master_xml.py

```diff
@@ -40,8 +40,8 @@ def getRawTooltips(locale):
             continue
         tooltipId = split[1].replace("tooltip_","").replace("_description","").replace("\"", "").lstrip("0")
 
-        # Remove any quotation marks, new lines and html tags.
-        rawTooltips[tooltipId] = cleanHtml(split[2].replace("\"", "").replace("\\n", "\n"))
+        # Remove any quotation marks and new lines.
+        rawTooltips[tooltipId] = split[2].replace("\"", "").replace("\\n", "\n")
```

Comment:
Fixed in [this](https://github.com/GwentCommunityDevelopers/gwent-data/pull/7/commits/8d46b9675797b2d357355202d0e6c8cc00b57089) commit.

Example output is now:
`"en-US": "<b><keyword=deploy>Deploy:</keyword></b> <keyword=spawn>Spawn</keyword> a Unicorn or a Chironex."`